pkg/logql/log/fmt.go

```diff
@@ -43,6 +43,14 @@ var (
 			r := regexp.MustCompile(regex)
 			return r.ReplaceAllLiteralString(s, repl)
 		},
+		"count": func(regexsubstr string, s string) int {
+			r := regexp.MustCompile(regexsubstr)
```

Comment:
This will panic if the regex is invalid
Please add a test case for that as well

Reply:
Tests have been added.